Clinical-Genomics/scout
# Side panel reports

Adds/removes the following icons to the case sidebar
- [x] general report html icon & link (#1648)
- [x] general report pdf icon & link (same issue as above
- [x] coverage report html icon & link (same issue as above)
- [x] coverage report pdf icon & link (same issue as above) **If it works just like this it's a miracle!! **
- [x] removes the general delivery report link whenever several reports are available (fix #1643)
- [x] mtDNA report has a clickable excel icon that makes you download the zipped report

**How to test**:
1. Install it on clinical-db stage
1. Go to a case with coverage report and possibly several analyses (example: https://scout-stage.scilifelab.se/cust002/F0000496)
1. Make sure that all the above icons are present with links that work 

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
